File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day05AlchemicalReductionTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day05AlchemicalReductionTest {
    @Test
    public void checkReal() {
        Day05AlchemicalReduction im = new Day05AlchemicalReduction("/twenty_eighteen/Day05-AlchemicalReduction-input.txt");

        assertEquals(11242, im.getPart1Answer().intValue());
        assertEquals(5492, im.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day05AlchemicalReduction im = new Day05AlchemicalReduction("/twenty_eighteen/Day05-AlchemicalReduction-example#1.txt");

        assertEquals(10, im.getPart1Answer().intValue());
        assertEquals(4, im.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day05AlchemicalReductionTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day05AlchemicalReductionTest  extends FlatSpec with Matchers {
  "2018 Day 05 - Input" should "calculate answers" in {
    val dp = new Day05AlchemicalReduction("/twenty_eighteen/Day05-AlchemicalReduction-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day05AlchemicalReduction("/twenty_eighteen/Day05-AlchemicalReduction-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
